serialization/python_pickle.ksy

```diff
@@ -0,0 +1,448 @@
+meta:
+  id: python_pickle
+  title: Python pickle serialization format (protocol 3)
```

Comment:
Since the Pickle data format is backwards-compatible, maybe the "protocol 3" part should be changed to "protocol 1 through 3".

Or perhaps this information should go into the `doc` block rather than the title. If the id or title of a spec mentions a specific format version, that usually indicates that there are other versions of the format that are incompatible enough to need their own spec. For the Pickle format this is not the case - because it is backwards-compatible, a single spec should be enough for all versions of the format, so there's no need to differentiate between versions in the title.

Reply:
> perhaps this information should go into the doc block

Agreed, modulo protocol 0 is the earliest version.